src/darker/black_diff.py

```diff
@@ -35,13 +35,15 @@
 
 import logging
 import sys
+from distutils.version import LooseVersion
```

Comment:
I wouldn’t use this class. its comparison operators just return whatever if a version doesn’t parse right.

These days I always use [`packaging.version`](https://packaging.pypa.io/en/latest/version.html).

I think #189 is a simpler fix here.